schema.sql

```diff
@@ -7,6 +7,25 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL(5, 2),
-    species VARCHAR(50),
+    species_id INT,
+    owner_id INT,
+    FOREIGN KEY(owner_id) REFERENCES owners(id) ON DELETE CASCADE,
+    FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADEs,
     PRIMARY KEY(id)
```

Comment:
- In this project, you are supposed to modify the animal's table as below and add commands in this `schema.sql` file.:

- [ ] Remove column species
- [ ] Add column species_id which is a foreign key referencing species table
- [ ] Add column owner_id which is a foreign key referencing the owners table

`Hint`: Think about `ALTER` command